vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/BasicUseIT.java

```diff
@@ -131,7 +132,24 @@ public void crudAndGeneratedGridReactToThemeVariantChanges() {
         Assert.assertNotEquals("no-border", grid.getAttribute("theme"));
     }
 
+    @Test
+    public void newEventItem() {
+        CrudElement crud = $(CrudElement.class).waitForFirst();
+        $("#newEventListener").onPage().first().click();
```

Comment:
As a general comment using `$("#newEventListener")` with a selector might work but it's not a supported case.
In this case  the implementation in `getTestButton` is preferable, you can use that helper instead.